docs/dev/table/hive/index.md

```diff
@@ -69,7 +69,7 @@ To integrate with Hive, users need the following dependencies in their project.
 
 <dependency>
   <groupId>org.apache.flink</groupId>
-  <artifactId>flink-hadoop-compatibility_{{site.version}}</artifactId>
+  <artifactId>flink-hadoop-compatibility{{site.scala_version_suffix}}</artifactId>
```

Comment:
please use `{{ site.scala_version_suffix }}`

Reply:
Thank you very much, I will revert it.